pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -62,6 +62,10 @@ const (
 	adminSecretName   = "admin-secret"
 	clusterSecretName = "cluster-name"
 
+	// configuration map for csi
+	csiConfigName = "ceph-csi-config"
```

Comment:
How about calling it `rook-ceph-csi-config`? Other rook CMs have the rook prefix.

Reply:
Done/obsoleted.